lib/plugins/rotate_asg_instances.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'aws-sdk'
+require_relative 'rotate_asg_instances_lib/asg'
+
+module Moonshot
+  module Plugins
+    # Rotate ASG instances after update.
```

Comment:
This comment seems to be already explained (yay readable code). Do we need it?

Reply:
This is community-accepted code style.  I think we should leave it.